tokio/src/net/udp/socket.rs

```diff
@@ -11,10 +11,11 @@ cfg_net! {
     ///
     /// UDP is "connectionless", unlike TCP. Meaning, regardless of what address you've bound to, a `UdpSocket`
     /// is free to communicate with many different remotes. In tokio there are basically two main ways to use `UdpSocket`:
-    ///     - one to many: [`bind`](`UdpSocket::bind`) and use [`send_to`](`UdpSocket::send_to`)
-    /// and `[`recv_from`](`UdpSocket::recv_from`) to communicate with many different addresses
-    ///     - one to one: [`connect`](`UdpSocket::connect`) and associate with a single address, using [`send`](`UdpSocket::send`)
-    /// and `[`recv`](`UdpSocket::recv`) to communicate only with that remote address
+    ///
+    /// * one to many: [`bind`](`UdpSocket::bind`) and use [`send_to`](`UdpSocket::send_to`)
+    /// and [`recv_from`](`UdpSocket::recv_from`) to communicate with many different addresses
+    /// * one to one: [`connect`](`UdpSocket::connect`) and associate with a single address, using [`send`](`UdpSocket::send`)
+    /// and [`recv`](`UdpSocket::recv`) to communicate only with that remote address
```

Comment:
I think this would make it clearer in the doc.
```suggestion
    /// * one to many: [`bind`](`UdpSocket::bind`) and use [`send_to`](`UdpSocket::send_to`)
    ///   and [`recv_from`](`UdpSocket::recv_from`) to communicate with many different addresses
    /// * one to one: [`connect`](`UdpSocket::connect`) and associate with a single address, using [`send`](`UdpSocket::send`)
    ///   and [`recv`](`UdpSocket::recv`) to communicate only with that remote address
```